requirements.txt

```diff
@@ -0,0 +1,3 @@
+python-telegram-bot==12.7
+jdatetime
+convert_numbers
```

Comment:
Thank you for adding the requirements file! 
Could you please pin to specific versions of these dependencies?

Reply:
My pleasure! :)
Of course.